airflow/config_templates/config.yml

```diff
@@ -368,6 +368,13 @@
       type: string
       example: ~
       default: "30"
+    - name: compress_serialized_dags
+      description: |
+        If True, serialized DAGs are compressed before writing to DB
```

Comment:
We should note that turning this on would (currently) break the DAG dependencies view.